scripts/build/yuri.cmake

```diff
@@ -8,7 +8,7 @@ set_cache_var(CELERITAS_BUILD_DOCS BOOL ON)
 set_cache_var(CELERITAS_USE_CUDA BOOL OFF)
 set_cache_var(CELERITAS_USE_Geant4 BOOL OFF)
 set_cache_var(CELERITAS_GIT_SUBMODULE BOOL OFF)
-set_cache_var(CELERITAS_USE_MPI BOOL ON)
+set_cache_var(CELERITAS_USE_MPI BOOL OFF)
```

Comment:
Is this change of default intentionally in this PR? (i.e. is disabling MPI needed to "fix SB possition..."?)

Reply:
Nope unintentional, will revert